kaldb/src/main/java/com/slack/kaldb/logstore/schema/SchemaAwareLogDocumentBuilderImpl.java

```diff
@@ -264,12 +264,17 @@ static String makeNewFieldOfType(String key, FieldType valueType) {
     return key + "_" + valueType.getName();
   }
 
-  private static void convertValueAndIndexField(
+  private void convertValueAndIndexField(
       Object value, FieldType valueType, LuceneFieldDef registeredField, Document doc, String key) {
     Object convertedValue =
         FieldType.convertFieldValue(value, valueType, registeredField.fieldType);
     if (convertedValue == null) {
-      throw new RuntimeException("No mapping found to convert value");
+      LOG.warn(
+          String.format(
```

Comment:
A better approach here would be to use the logging token replacements, with the `{}` instead of a string format.